js/admin/notifications.js

```diff
@@ -54,7 +54,9 @@ $(document).ready(function() {
   });
 
 	// call it once immediately, then use setTimeout
-	getPush();
+  if (parseInt(show_new_orders) || parseInt(show_new_customers) || parseInt(show_new_messages)) {
```

Comment:
Check this link: https://github.com/PrestaShop/PrestaShop/pull/7955/files ; you will see it more easier.

It's set in the initNotifications() method,